src/css/styles/_c-banner.scss

```diff
@@ -0,0 +1,82 @@
+.c-banner {
+  position: relative;
+  width: 100%;
+  min-height: 568px;
+  max-height: 1440px;
+  margin: 0 auto;
+  margin-bottom: 6rem;
+  background: url("../img/c-banner__bg.jpg") no-repeat 40% 60%;
+  background-size: auto 130%;
+
+  &__title {
+    margin-bottom: 3rem;
+    font-size: 4rem;
+    line-height: 120%;
+    text-shadow: 0 0 10px $col-bg;
+  }
+
+  &__txt {
+    margin-bottom: 4rem;
+    font-size: 1.4rem;
+    line-height: 120%;
+    text-shadow: 0 0 10px $col-bg;
+  }
+
+  &__btn {
+    margin: 0 auto;
+  }
+
+  @include media-breakpoint-down(lg) {
+    background-position: 65% 100%;
+    background-size: auto 120%;
+
+    &__title {
+      margin-bottom: 3rem;
+      font-size: 3.3rem;
+    }
+
+    &__txt {
+      margin-bottom: 3rem;
+      font-size: 1.2rem;
+    }
+  }
+
+  @include media-breakpoint-down(md) {
+    &__title {
+      margin-bottom: 2rem;
+      font-size: 2.5rem;
+    }
+
+    &__txt {
+      margin-bottom: 3rem;
+      font-size: 1rem;
+    }
+  }
+
+  @include media-breakpoint-down(xs) {
+    margin-bottom: 4rem;
+    background-position: 66% 100%;
+
+    &__hero {
+      margin: 0 1rem;
+    }
+
+    &__title {
+      margin-bottom: 1rem;
+      font-size: 2rem;
+    }
+
+    &__txt {
+      margin-bottom: 1.5rem;
+      font-size: 0.8rem;
```

Comment:
![image](https://user-images.githubusercontent.com/789841/50002020-0a26d480-ff9f-11e8-9cc2-d0ea8ae19d2d.png)

- [ ] font-size on mobile

Reply:
Is it 0.9rem? Looks better on most mobiles, on the smallest text part can be slightly too big, but still- it's worth the change. Now on mobiles it's:  `font-size: 0.9rem;` and `line-height: 140%;`

Reply:
it is 1rem. IMHO 0.9 is still to small